lib/instrumentation/noop-transaction.js

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and other contributors where applicable.
+ * Licensed under the BSD 2-Clause License; you may not use this file except in
+ * compliance with the BSD 2-Clause License.
+ */
+
+'use strict';
+
+const constants = require('../constants');
+
+module.exports = NoopTransaction;
```

Comment:
nit: Would you object to this being a named export, rather than a default export? I know that existing files of ours use default exports, but I've been trying to move away from them. I don't have a strong argument against default exports other than when mixing CommonJS and ESM, default exports can be problematic. Obviously that argument doesn't apply in this case.

```suggestion
module.exports = {
  NoopTransaction
};
```

nit(style): My preference is for exports to be listed at the bottom. I guess someday I'll need to adapt to ESM-style `export ...` statements throughout the module. :)

Reply:
Done in commit cf60004d